src/comms/models.py

```diff
@@ -14,6 +14,15 @@
 __maintainer__ = "Birkbeck Centre for Technology and Publishing"
 
 
+class ActiveNewsItemManager(models.Manager):
+    def get_queryset(self):
+        return super().get_queryset().filter(
```

Comment:
This could also be written using a single filter with Q:

```
return super().get_queryset().filter(
    (models.Q(start_display__lte=timezone.now()) | models.Q(start_display=None)) &
    (models.Q(end_display__gte=timezone.now()) | models.Q(end_display=None))
)
```